voorhoede/tudelft-spacefinder
# Use strongly recommended ESLint Vue ruleset

# Associated issue
https://trello.com/c/18mmmchA/34-use-strongly-recommended-eslint-vue-ruleset


# How to test
run linting task or see https://github.com/voorhoede/tudelft-spacefinder/actions/runs/4491120353/jobs/7899127789?pr=167#step:7:1



# Checklist
